src/test/java/Homework21.java

```diff
@@ -0,0 +1,51 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Action;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Homework21 extends BaseTest {
+
+    @Test
+    public void renamePlaylist() {
+
+        String updatedPlaylistSuccessMsg = "Updated playlist \"Sample Edited Playlist.\"";
+
+
+        provideEmail("clemence.breslin@testpro.io");
+        providePassword("0usTFBYn");
+        logintoKoel();
+
+        doubleClickPlaylist();
+        enterNewName("New Playlist");
+        Assert.assertEquals(getRenamePlaylistSuccessMsg(), updatedPlaylistSuccessMsg);
+    }
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   For example
   
   
![Clean_Test](https://github.com/theythemclemence/A55-QA-Automation-Framework/assets/104761482/b69c7f46-eb9f-4c2f-836e-b06cc529be64)